ocs_ci/templates/lvmo-deployment/lvms-cluster.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: lvmscluster
+  namespace: openshift-storage
+spec:
+  storage:
+    deviceClasses:
+      - name: vg1
+        thinPoolConfig:
+          name: thin-pool-1
+          overprovisionRatio: 10
```

Comment:
we usually add `sizePercent: 90` here, not sure if its mandatory

Reply:
Not mandatory

Reply:
@AviadP  This file is a clone of [lvm-cluster-default.yaml](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/templates/lvmo-deployment/lvm-cluster-default.yaml) and it doesn't has `sizePercent:` parameter. should i update both the files?